defi/amms/bonding/tests/lib.rs

```diff
@@ -27,7 +27,7 @@ fn setup_fixture<'a, L: Ledger>(env: &mut TestEnv<'a, L>) -> (User, User, Resour
 
     const PACKAGE: &str = "bonding";
 
-    env.publish_package(PACKAGE, include_code!());
+    env.publish_package(PACKAGE, include_code!("out"));
```

Comment:
Don't forget to rename this when you remove the `name="out"`

Reply:
pushed another commit removing the lib name override and fixing this line to match.

Reply:
Thanks !